### PR TITLE
Stop logging /cluster.ClusterService/Create

### PR DESCRIPTION
**Pull request by dthomson25**

### CODE REVIEW

**Review by alexmt — APPROVED** [review]
LGTM

**Comment by jessesuen** [comment]
@dthomson25 the other places of concern are here:
* https://github.com/argoproj/argo-cd/blob/master/cmd/argocd-util/main.go#L157
and here:
* https://github.com/argoproj/argo-cd/blob/master/util/kube/ctl.go#L279
Secrets need to be redacted from the output.

File: server/server.go [hunk 1]
```diff
@@ -395,6 +395,7 @@ func (a *ArgoCDServer) newGRPCServer() *grpc.Server {
 		grpc.ConnectionTimeout(300 * time.Second),
 	}
 	sensitiveMethods := map[string]bool{
+		"/cluster.ClusterService/Create":         true,
```
Comment (jessesuen) [inline server/server.go]: Update should be added here as well.

**Review by alexmt — APPROVED** [review]
LGTM

**Comment by alexmt** [comment]
Please fix linter before merging:

```

gometalinter.v2 --config gometalinter.json ./...
cmd/argocd-util/main.go:177:24:warning: ineffectual assignment to err (ineffassign)
controller/appcontroller_test.go:1::warning: file is not gofmted with -s (gofmt)
Makefile:96: recipe for target 'lint' failed
make: *** [lint] Error 1
```

**Pull request merged**